### PR TITLE
v3.1.0 update - create package

### PR DESCRIPTION
**Pull request by premanshup**
v3.1.0
- Improvement: Redesigned the entire customizer UI for better user experience. ( https://wpastra.com/docs/revamped-astra-customizer/ )
- Improvement: Added design options for Builder - Toggle Menu in Transparent header.
- Improvement: Optimized dynamic CSS functionality to reduce unwanted CSS on the page.
- Improvement: Overall HTML Structure improvement which includes removing blank spaces and removing unwanted HTML Tags.
- Improvement: Optimized customizer by removing PHP controls.
- Improvement: Optimized the SVG icons.
- Improvement: Builder - Transparent Header - Placeholder Text and Color options for Search.
- Improvement: Builder - Social Icons - Ability to add an extra Phone and Contact icons.
- Improvement: Builder - Social Icons - Added ability to select icon from an Icon Picker for individual social elements
- Improvement: Builder - Social Icons - Added a color option for labels of social icons.
- Improvement: Builder - Off-Canvas - Added an option to manage spacing between inner elements of the Off-Canvas.
- Improvement: Added new Page meta options for Above, Below and Mobile header.
- Fix: Footer vertical alignment option not not working.
- Fix: Above Footer & Below Footer showing up an unwanted border in the customizer preview.
- Fix: Updated Learn More Links on Astra Options page for Nav Menu & Easy Digital Downloads addon.
- Fix: Overflowing content not wrapping as per grid layout on responsive device for Header Footer builder layout.
- Fix: Builder - Transparent Header - Social icon, label custom color & official color not working.
- Fix: Builder - Transparent Header - Menu & Submenu style applying to Off-Canvas Menu.
- Fix: Block Cover align full CSS compatibility in Block editor with sidebar.
- Fix: Color picker icon not visible on hover.
- Fix: Font size not reverted back to the initial size when the value is deleted.
- Fix: Menu active link style not working when sub-menu item is active.
- Fix: Menu line height not being applied in frontend.
- Fix: Global Button Border and Typography controls are not working in the customizer preview.
- Fix: Breadcrumb position misalignment issue for inside header option.
- Fix: Search box misaligned in Dropdown option for Off-Canvas in Mobile.
- Fix: Submenu divider option adding heavy border around menu items on responsive devices.
- Fix: JS console error with Elementor – Header, Footer & Blocks plugin.
- Fix: Console error in customizer due to third party plugin dependency.
- Fix: Menu font size not working with old Header.
- Fix: Transparent Header - Menu link, background and hover colors not working for old header.
- Fix: Builder - Menu - Submenu container get hides when hover in between Top Offset spacing.

### CODE REVIEW

File: inc/customizer/extend-custom-controls/package-lock.json [hunk 1]
```diff
@@ -2764,34 +2764,11 @@
         "webpack-livereload-plugin": "^2.3.0"
```
Comment (Nikschavan) [inline inc/customizer/extend-custom-controls/package-lock.json:2764]: checking if you have run `npm audit fix`? there is a low severity dependency update being reported by dependbot which can be included in the update.

**Review by Nikschavan — APPROVED** [review]
Added one comment, Rest looks good to me. Let's 🚢

File: inc/customizer/extend-custom-controls/package-lock.json (hunk 1 above)
Comment (premanshup) [reply inc/customizer/extend-custom-controls/package-lock.json:2764]: @Nikschavan No! I will run and update it right away, thank you for sharing this.

Comment (premanshup) [reply inc/customizer/extend-custom-controls/package-lock.json:2764]: Done!

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**